civihr_employee_portal/src/Webform/WebformTransferService.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Webform;
+
+/**
+ * Responsible for ensuring reliable transfer of webforms between systems.
+ *
+ * Handles adding metadata for fields that rely on the database IDs and then
+ * using the metadata to use the correct IDs on whichever system the webform is
+ * exported to.
+ */
+class WebformTransferService {
+
+  /**
+   * @param \stdClass $node
+   */
+  public static function preExport(\stdClass $node) {
```

Comment:
Any reason why these methods are not actual class methods including the methods in the `WebformTransferConvertor` interface? Static methods are somewhat ugly. :(

Reply:
I would like to use non-static class methods, but since this is the employee portal I don't have any DI container available. It's also messy instantiating classes everywhere, especially if the dependencies change later. I'm not sure how to get around this, in CiviHR we can use the Symfony container, but just not here.